### DISCUSSION TITLE
Duda sobre ids en inputs

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#137] [Tarea 2] [answered]
Hola! tengo una duda con respecto a los ids de los pinguinos en los archivos input. Al igual que en la tarea anterior en las N lineas que representan a los pinguinos, los ids irán creciendo consecutivamente, es decir,  que en la primera linea el id sea 0, en la segunda 1 y asi sucesivamente, o podría suceder que los ids no sean correlativos? Quedo atento, muchas gracias!

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, no serán correlativos, pronto subiremos los tests y quedará más claro. Saludos!

**Reply by IgnacioHqz** [reply]
Perfecto, muchas gracias!
